astroid/exceptions.py

```diff
@@ -256,6 +261,18 @@ class InferenceOverwriteError(AstroidError):
     """
 
 
+class ParentMissingError(AstroidError):
```

Comment:
Can you add it here as well, just so it's documented?
https://github.com/PyCQA/astroid/blob/main/doc/api/astroid.exceptions.rst